internal/encryption/targets_test.go

```diff
@@ -52,7 +52,7 @@ func TestBaseEncryption_buildTargetMethods(t *testing.T) {
 					method = method.aes_gcm.example
 				}
 			`,
-			wantErr: `Test Config Source:3,25-32: Unsupported attribute; This object does not have an attribute named "static".`,
+			wantErr: `Test Config Source:3,13-38: Undefined Key Provider; Key provider static.basic is missing from the encryption configuration.`,
```

Comment:
`suggestion`
It would be great to have the test `TestBaseEncryption_buildTargetMethods` renamed as well.

Reply:
Fixed in a318008b3c :+1: